### ISSUE TITLE
[Feature Request] Adding a workflow to run basic tests 

### ISSUE THREAD

**Issue by shankarpriyank** [JunkFood02/Seal#149]
### Checklist

- [ ] This feature I'm requesting is already implemented in yt-dlp.
- [ ] This feature is merely a UI/UX update.
- [ ] This feature is suitable for primary users with little knowledge about yt-dlp.
- [ ] This feature is available for most websites, not only the video platform I use.
- [X] This feature is suitable for a large variety of videos.
- [X] This feature is not going to conflict with many of the existing options.

### Describe the solution you'd like

We can add a workflow to run some basic tests so that a new pull request does not end up breaking the application

**Comment by shankarpriyank** [comment]
@JunkFood02 I would be happy to add the workflow.
For the v0 of , we can do something like this [ sample ](https://github.com/shankarpriyank/DocDelivery/blob/master/.github/workflows/build.yml)

**Comment by JunkFood02** [comment]
I think we've already had a [GitHub Action thing](https://github.com/JunkFood02/Seal/blob/main/.github/workflows/android.yml) to do this? I know little about this tho

**Comment by shankarpriyank** [comment]
I had a look at that file, that is not exactly what I am talking about, the current workflow comes into action only after a new pr has been merged for example, if a pr breaks the app or some functionality the workflow will not allow that pr to be merged.

**Comment by JunkFood02** [comment]
For testing it I've request a PR from Hosted Weblate #150. It seems GitHub is running [the test](https://github.com/JunkFood02/Seal/runs/7978912583)

**Comment by shankarpriyank** [comment]
Btw this project seems interesting to me would like to collaborate.
